tracim/tracim/controllers/content.py

```diff
@@ -225,6 +234,15 @@ def get_one(self, file_id, revision_id=None):
                                  revision_id)
             preview_urls.append(url)
 
+        pdf_ready_mimetypes = []  # type: List[str]
```

Comment:
I did move this decision-making part of code into preview-generator. See https://github.com/algoo/preview-generator/commit/205cb460fdbee33d9e701f5024be6a5c22319b5c

So instead, use manager.has_pdf_preview(file_path) which returns True or False according to the availability of PDF previewing"